ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1277 @@
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "gAL9Y4VTOLT1"
+      },
+      "source": [
+        "## Prepare for output\n",
+        "\n",
+        "### Step 1. Create dataset for output\n",
+        "\n",
+        "You need a BigQuery dataset to host the output data in `us-central1`. Input the name of the dataset you want to created and specify the name of the table you want to store the output later. These will be used later in the notebook.\n",
+        "\n",
+        "**Make sure that the table name does NOT already exist**.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "4Oc-jrd6Ow7N"
+      },
+      "source": [
+        "from google.cloud import bigquery"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "QOaGi2PrEAwA"
+      },
+      "source": [
+        "# Output dataset, as created in the step 1.\n",
+        "DESTINATION_DATA_SET = \"movie_prediction\"  #@param {type:\"string\"}\n",
+        "# Output table. Make sure that the table does NOT already exist; the BatchReadFeatureValues API cannot overwrite an existing table\n",
+        "DESTINATION_TABLE_NAME = \"training_data\" #@param {type:\"string\"}\n",
+        "\n",
+        "DESTINATION_PATTERN = \"bq://{project}.{dataset}.{table}\"\n",
+        "DESTINATION_TABLE_URI = DESTINATION_PATTERN.format(project=PROJECT_ID,\n",
+        "    dataset=DESTINATION_DATA_SET, table=DESTINATION_TABLE_NAME)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "RKhmymT-O0vy"
+      },
+      "source": [
+        "# Create dataset\n",
+        "client = bigquery.Client()\n",
+        "dataset_id = \"{}.{}\".format(client.project, DESTINATION_DATA_SET)\n",
+        "dataset = bigquery.Dataset(dataset_id)\n",
+        "dataset.location = \"us-central1\"\n",
+        "dataset = client.create_dataset(dataset, timeout=30)\n",
+        "print(\"Created dataset {}.{}\".format(client.project, dataset.dataset_id))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XoEqT2Y4DJmf"
+      },
+      "source": [
+        "### Import libraries and define constants"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "isNzmylQXjly"
+      },
+      "source": [
+        "# Other than project ID, location and featurestore ID and endpoints needs to be set\n",
+        "LOCATION = \"us-central1\"  #@param {type:\"string\"}\n",
+        "API_ENDPOINT = \"us-central1-aiplatform.googleapis.com\"  #@param {type:\"string\"}\n",
+        "INPUT_CSV_FILE = \"gs://cloud-aiplatform-featurestore-us-central1/read_entity_instance.csv\""
```

Comment:
Unless I did something goofy I don't think INPUT_CSV_FILE is the right file?
```
gsutil cat "gs://cloud-aiplatform-featurestore-us-central1/read_entity_instance.csv"|head -20 
bikes,stations,timestamp
097G,3841,2019-10-27T08:28:14Z
057G,2565,2019-10-19T08:58:56Z
2266,2707,2019-10-15T09:24:11Z
```

Reply:
this is a bug, fixed